assets/css/common/post-single.css

```diff
@@ -268,7 +268,7 @@
     padding: 0.4em;
 }
 
-.dark .toc {
+[data-theme="dark"] .toc {
```

Comment:
We can use `background: light-dark(var(--code-bg), var(--entry))` instead to avoid setting the same thing twice in both `[data-theme="dark"]` and `@media (prefers-color-scheme: dark)`

Reply:
it's not same thing twice. `@media` is used only when user has js disabled.

Reply:
Yeah I know there were 2 cases, but with `light-dark` you can write it once and cover the 2 cases